account_cutoff_accrual_base/views/res_config_settings.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+      <field name="name">res.config.settings.view.form.inherit.account</field>
+      <field name="model">res.config.settings</field>
+      <field name="priority" eval="50"/>
+      <field name="inherit_id" ref="account_cutoff_base.res_config_settings_view_form"/>
+      <field name="arch" type="xml">
+        <xpath expr="//div[@name='account_closing_cutoffs']" position="inside">
+
+          <div class="col-xs-12 col-md-12 o_setting_box">
+            <div class="o_setting_left_pane">
+            </div>
+            <div class="o_setting_right_pane">
+              <label string="Accruals"/>
+              <div class="content-group">
+                <div class="row mt16">
+
+                  <label string="Account for Accrued Expense" class="col-md-3 o_light_label"/>
```

Comment:
I think you can use
`<label for="default_accrued_expense_account_id" class="col-md-3 o_light_label"/>`
to avoid to redifine the string and use the string defined into the model.
Same remark for 4 fields.

Reply:
You are correct, so I changed it.
I had to make the label size a little bigger so it would not overflow as the strings defined on the model all start with 'Default' which seems kind of superfluous to me.